vmdb/app/controllers/miq_request_controller.rb

```diff
@@ -91,10 +97,19 @@ def request_edit
       @req_id = params[:id]
       @prov_type = prov.resource_type
     else
-      session[:checked_items] = prov.options[:src_ids]
-      @refresh_partial = "reconfigure"
-      @_params[:controller] = "vm"
-      reconfigurevms
+      if prov.kind_of?(MiqProvisionConfiguredSystemRequest)
+        @org_controller = "configured_system"
```

Comment:
Can you change this to `@org_controller = prov.org_controller`?  Also do the same thing a few lines up.